content/applications/services/project/tasks/task_stages_statuses.rst

```diff
@@ -0,0 +1,71 @@
+========================
+Task stages and statuses
+========================
+
+Task stages
+===========
+
+Task stages are displayed as columns in the project's Kanban view, and allow you to update the
+progress of its tasks with a drag-and-drop. In most projects, the stages will be akin to **New**,
+**In progress**, **Backlog**, etc.
+
+By default, task stages are project-specific but can be shared across multiple projects that follow
+the same workflow.
+
+Creating task stages
+--------------------
+
+Odoo Project doesn’t provide default stages but instead allows you to create custom stages tailored
+to your specific business needs. You are prompted to do so immediately after creating a new project.
+To create a stage, type its name into the :guilabel:`Stage...` field, then click :guilabel:`Add`.
+
+.. tip::
+   Click :guilabel:`See examples` to find ideas for stage names applicable to your line of business.
+
+Editing task stages
+-------------------
+
+To edit the task stage, click the :icon:`fa-cog` (:guilabel:`cog`) icon next to its name. From
+there, click one of the following:
+
+ - :guilabel:`Fold`: to hide the task stage and all of the tasks in this stage from the Kanban view.
+ - :guilabel:`Edit`:
+
+   - :guilabel:`Name`: to change the name of the stage.
+   - :guilabel:`SMS/Email Template`: to automatically send an email or SMS notification to the
+     customer when a task reaches this stage.
+   - :guilabel:`Folded in Kanban`: to hide the task stage and all of the tasks in this stage from
+     the Kanban view.
+   - :guilabel:`Projects`: to share this task stage between several projects.
+   - :guilabel:`Automations`: to create :doc:`custom rules that trigger automatic actions
+     <../../../studio/automated_actions>` (e.g.: the creation of activities, adding followers, or
```

Comment:
```suggestion
     <../../../studio/automated_actions>` (e.g., creating activities, adding followers, or
```
comma instead of colong after e.g. (as per Grammarly) + -ing form for creating since the other ones are -ing form)